src/components/AccountSelector.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import {
+  FormControl,
+  InputLabel,
+  makeStyles,
+  MenuItem,
+  Select,
+} from "@material-ui/core";
+import AccountData from "../datamodels/account-model";
+
+const useStyles = makeStyles(() => ({
+  selectField: {
+    minWidth: 120,
+    maxWidth: 250,
+  },
+}));
+
+const AccountSelector = (props) => {
+  const classes = useStyles();
+  let { name, accounts, value } = props;
+  return (
+    <FormControl>
+      <InputLabel>Account</InputLabel>
+      <Select
+        className={classes.selectField}
+        name={name}
+        value={value}
+        onChange={props.onChange}
+      >
+        <MenuItem value="" />
+        {accounts.map((account) => (
+          <MenuItem key={account.id} value={account.id}>
```

Comment:
Could you explain why we need a key in arrays elements?
Can we use key on other elements too?